Add a reminder on the supported languages for Stack Trace Linking (#5642)
getsentry/sentry-docs: src/docs/product/integrations/source-code-mgmt/github/index.mdx

### Stack Trace Linking

<Note>

> Only JavaScript and Python are supported at this moment.

Reviewer: ```suggestion
This feature is currently only supported for JavaScript and Python.
```
